vlmd-submission-tools/tests/test_subcommand_upload_dictionary_to_mds.py

```diff
@@ -66,7 +69,9 @@ def test_upload_dictionary_to_mds(self, mocked_mds_put, mocked_mds_post, mocked_
         new_metadata = {
             "_guid_type": "discovery_metadata",
             "gen3_discovery": "discovery_metadata",
-            "data_dictionaries": new_data_dictionaries
+            "variable_level_metadata": {
+                "data_dictionaries": new_data_dictionaries
```

Comment:
might be good to add another test case or update one of these existing test cases in here which has a `"common_data_elements"` section inside `"variable_level_metadata"`, to demo the content of that section is not being altered by the DD submission scripts